packages/shared/src/lib/query.ts

```diff
@@ -10,11 +10,22 @@ import { EmptyObjectLiteral } from './kratos';
 import { LoggedUser } from './user';
 import { FeedData, Post, ReadHistoryPost } from '../graphql/posts';
 import { ReadHistoryInfiniteData } from '../hooks/useInfiniteReadingHistory';
+import { MainFeedPage } from '../components/utilities';
```

Comment:
I'm not a big fan of the naming, it's a bit unclear for me even what is a main feed page and what is not, but it has nothing to do with this PR so will not mark it as fix.

Reply:
Right. I agree with that. Probably SharedFeedPage since those are the shared feeds with the extension.

Reply:
Should be refactored here https://github.com/dailydotdev/apps/pull/2289/commits/c92c2df1f7af7d5f1144fb7afb1d7b0b10abf7f5